lib/bootloader_setup.pm

```diff
@@ -265,6 +265,8 @@ sub boot_local_disk {
 sub boot_into_snapshot {
     send_key_until_needlematch('boot-menu-snapshot', 'down', 10, 5);
```

Comment:
Why not wait the boot-menu shown then send key? In fact the send_key_until_needlematch will send key firstly, then check the screen, so this may cause can't match the tag if it is on the first line.

Reply:
https://openqa.suse.de/tests/8184453#step/grub_test_snapshot/8
 boot-menu-snapshot is another page not same with grub page

Reply:
Ok, so the code is just what I expected. Please run the VRs on other ARCHes also.